docs/docs/adapters/edgedb.md

```diff
@@ -0,0 +1,273 @@
+---
+id: edgedb
+title: EdgeDB
+---
+
+# EdgeDB
+
+To use this Adapter, you need to install `edgedb`, `@edgedb/generate`, and the separate `@next-auth/edgedb-adapter` package:
+
+```bash npm2yarn2pnpm
+npm install next-auth edgedb @next-auth/edgedb-adapter
+npm install @edgedb/generate --save-dev
```

Comment:
Now you're using raw edgeql queries, I don't think `@edgedb/generate` is needed any more right?

Reply:
We're using raw edgeql queries for the adapter itself, but I think we should still recommend developers to use `@edgedb/generate` in their code - right? As per: https://www.edgedb.com/docs/guides/tutorials/nextjs#generating-the-query-builder

In other words.. this documentation is recommending developers to install `@edgedb/generate` to their own code (along with the newly-created `@next-auth/edgedb-adapter`)

You can see how I've added this to the "demo" app here: https://github.com/nextauthjs/next-auth/pull/5781/files/97c9e801ea6d925d77cbb1846675407aeb26ceb7#diff-9ba7e4d9762adead1effbf9a9a1a94b85191064e2f9786da86b7918a7629e27d